ci/container/test_component.sh

```diff
@@ -102,6 +102,27 @@ if [[ -z "$GITHUB_ACTIONS" ]]; then
     fi
 fi
 
+echo "[INFO] Build is using node versions"
+npm version
+
+set +e
+npm version | grep "node: '12"
+usingNode12=$?
+set -e
+if [ $usingNode12 -eq 0 ]
+then
+  echo "[DEBUG] We are using node v12"
+else
+  echo "[DEBUG] Installing newer node 12"
+  export NVM_DIR=`pwd`/nvm
+  cp -r /usr/local/nvm $NVM_DIR
+  source $NVM_DIR/nvm.sh
+  nvm install 12
+
+  echo "[INFO] Build is using node versions"
+  npm version
+fi
+
```

Comment:
That change is not in scope but it is your choice to add this in the same PR.

Reply:
it is in separate PR https://github.com/snowflakedb/snowflake-connector-nodejs/pull/573 which should be merged first